internal/controller/clientprofilemapping_controller.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
any specific reason for this change?

Reply:
I can see this happening with many PRs. I think something in the template changed.

Reply:
Let me check this after this PR